src/components/TodoItem/TodoItemStyled.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+export const TodoItemContainer = styled.div`
+  display: flex;
+  background-color: white;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: solid 1px gray;
+  width: 500px;
+  height: 11px;
+  margin: 0 auto;
+  padding: 20px 30px;
+`;
+
+export const TodoCheker = styled.div`
+  justify-content: center;
+  width: 20px;
+  height: 20px;
+  border: solid 1px gray;
+  border-radius: 50%;
+  cursor: pointer;
+  background-color: ${({ $completed }) => ($completed ? '#c610d5' : 'white')};
+
+  &:hover {
+    border: solid #c610d5 2px;
+  }
+`;
+
+export const TodoTitle = styled.p`
+  color: black;
+  font-family: '-moz-initial';
+  font-size: 16px;
+  margin: auto;
+  cursor: pointer;
+  text-decoration: ${({ $completed }) => ($completed ? 'line-through' : ' ')};
+  opacity: ${({ $completed }) => ($completed ? '0.5' : '1')};
+`;
```

Comment:
You can also do it this way:

const Container = styled.p(props => ({
  color: 'black',
  font-family: '-moz-initial',
  font-size: 16,
  margin: 'auto',
  cursor: 'pointer',
  text-decoration: props.completed  ? 'line-through'  :  ' ',
  opacity: props.completed ? 0.5 : 1,
}))

And this way:

const Container = styled.p(({completed}) => ({
  color: 'black',
  font-family: '-moz-initial',
  font-size: 16,
  margin: 'auto',
  cursor: 'pointer',
  text-decoration: completed  ? 'line-through'  :  ' ',
  opacity: completed ? 0.5 : 1,
}))

Try them!